VideoLocker/src/main/java/org/edx/mobile/social/SocialLoginDelegate.java

```diff
@@ -211,21 +213,25 @@ public ProfileModel call( ) {
 
                 // do SOCIAL LOGIN first
                 AuthResponse social = null;
+                HashMap<String, CharSequence> descParams = new HashMap<>();
+                String platformName = environment.getConfig().getPlatformName();
+                descParams.put("platform_name", environment.getConfig().getPlatformName());
+                descParams.put("platform_destination", environment.getConfig().getPlatformDestinationName());
```

Comment:
I guess it’s fine as it is for now - if we find that it’s being duplicated multiple times later, then we can provide a utility method at that point.
> On 20-Oct-2015, at 3:48 AM, Akiva Leffert <notifications@github.com> wrote:
> 
> In VideoLocker/src/main/java/org/edx/mobile/social/SocialLoginDelegate.java <https://github.com/edx/edx-app-android/pull/397#discussion_r42436100>:
> 
> > @@ -211,21 +213,25 @@ public ProfileModel call( ) {
> >  
> >                  // do SOCIAL LOGIN first
> >                  AuthResponse social = null;
> > +                HashMap<String, CharSequence> descParams = new HashMap<>();
> > +                String platformName = environment.getConfig().getPlatformName();
> > +                descParams.put("platform_name", environment.getConfig().getPlatformName());
> > +                descParams.put("platform_destination", environment.getConfig().getPlatformDestinationName());
> Err, I mean, it is specific to a small number of strings. But if you have a suggestion for a place to put it, I'm happy to put it there.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/edx/edx-app-android/pull/397/files#r42436100>.
>